projects/plugins/jetpack/_inc/client/discussion/subscriptions.jsx

```diff
@@ -134,6 +141,27 @@ class SubscriptionsComponent extends React.Component {
 									'jetpack'
 								) }
 							/>
+							{ this.props.isSubscriptionModalEnabled && (
+								<>
+									<ToggleControl
+										checked={ isSubscriptionsActive && this.props.getOptionValue( 'sm_enabled' ) }
+										disabled={
+											! isSubscriptionsActive ||
+											unavailableInOfflineMode ||
+											this.props.isSavingAnyOption( [ 'subscriptions' ] )
+										}
+										toggling={ this.props.isSavingAnyOption( [ 'sm_enabled' ] ) }
+										onChange={ this.handleSubscribeModalToggleChange }
+										label={ __( 'Enable subscriber modal', 'jetpack' ) }
+									/>
+									<p className="jp-form-setting-explanation">
```

Comment:
I used separate `p` here because `help={}` prop just doesn't look great at all. I copied the pattern from elsewhere in Jetpack settings:

**help prop**
<img width="1091" alt="Screenshot 2023-07-13 at 11 38 34" src="https://github.com/Automattic/jetpack/assets/87168/062405a8-0727-447f-ac52-164af82c587f">

**paragraph**
<img width="1078" alt="Screenshot 2023-07-13 at 11 44 01" src="https://github.com/Automattic/jetpack/assets/87168/43939d26-dafd-41a9-9311-5a90566af154">